README.md

```diff
@@ -47,36 +47,18 @@ During this phase, 18F will provide GSA OSC with the following:
 
 Local development is based heavily on [CircleCI's Docker + Drupal deployment guide](https://circleci.com/blog/continuous-drupal-p1-maintaining-with-docker-git-composer/).
 
-Once you've pulled down this repo, `cd` into it, and then, 
-from the root of the directory, run:
-
-```docker-compose up -d --build```
-
-which tells Docker Compose to do a fresh build of the enclosed Dockerfile.
-
+1. Pull down this repo
+1. `cd` into the `app` directory
```

Comment:
Why are you going into the app directory then backing up again?

Reply:
The `composer` file lives in app while the `Dockerfile` lives at the root. You think it'd be ok to move the `Dockerfile` into app?

Reply:
I guess i"m just confused on how the prior instructions weren't working, and why the extra dip into `app` is needed.

Reply:
there were 2 sets of composer files previously - one set at the root and one set inside `app`. The composer files at the root were removed in favor of having all things Drupal inside of `app` since we are also using this repo for weekly ship notes. 

The Dockerfile at the root were drops you into the app folder when you ran the following
>Then run docker exec -it osc-website-pa_drupal_1 bash, which starts the container, opens a bash shell, and drops you into the app directory. 

Within the dockerfile the WORKDIR is set to `/app`